src/util/generate/metadata.ts

```diff
@@ -0,0 +1,97 @@
+export const internal = {
+  upload: 'upload',
+  ignore: 'ignore',
+  destructure: 'destructure',
+  manual: 'manual'
+}
+export const builders = {
+  static: '@now/static',
+  staticBuild: '@now/static-build',
+  next: '@now/next',
+  node: '@now/node',
+  nodeServer: '@now/node-server',
+  rust: '@now/rust',
+  markdown: '@now/md',
+  mdxDeck: '@now/mdx-deck',
+  go: '@now/go',
+  php: '@now/php',
+  bash: '@now/bash',
+  python: '@now/python'
+}
+
+export const extensions: {
+  [key: string]: string[]
+} = {
+  '.js': [
+    builders.node,
+    builders.nodeServer,
+    builders.static
+  ],
+  '.ts': [
+    builders.node,
+    builders.nodeServer
+  ],
+  '.html': [builders.static],
+  '.htm': [builders.static],
+  '.css': [builders.static],
+  '.rs': [builders.rust],
+  '.md': [builders.markdown],
+  '.markdown': [builders.markdown],
+  '.mdx': [builders.mdxDeck],
+  '.go': [builders.go],
+  '.php': [builders.php],
+  '.sh': [builders.bash],
+  '.py': [builders.python]
+}
+
+export const locale: {
+  [key: string]: {
+    single: string,
+    many: string
+  }
+} = {
+  [builders.static]: {
+    single: 'This is a public static file',
+    many: 'These are public static files'
+  },
+  [builders.staticBuild]: {
+    single: 'This file needs to be built as static',
+    many: 'These files need to be built as static'
+  },
+  [builders.next]: {
+    single: 'This is a Next.js project',
+    many: 'This is a Next.js project'
+  },
+  [builders.node]: {
+    single: 'This is a Node.js lambda',
+    many: 'Each of these are indivudual Node.js endpoints'
+  },
+  [builders.nodeServer]: {
+    single: 'This is Node.js app listening on a port',
+    many: 'These form a Node.js app listening on a port'
+  },
+  [builders.mdxDeck]: {
+    single: 'This is an MDX Deck Slide',
+    many: 'These are MDX Deck Slides'
+  },
+  'gatsby': {
```

Comment:
Interesting, you added a gatsby option but we don't have a gatsby builder?

Reply:
https://github.com/zeit/now-cli/pull/2276/files#diff-522f0c1cbb215e72b1d414b38210dcadR19

Reply:
Amazing 🤩